io_scene_niftools/modules/nif_export/geometry/mesh/__init__.py

```diff
@@ -460,6 +461,24 @@ def export_tri_shapes(self, b_obj, n_parent, n_root, trishape_name=None):
 
                     if NifData.data.version >= 0x04020100 and NifOp.props.skin_partition:
                         NifLog.info("Creating skin partition")
+
+                        # warn on bad config settings
+                        if game == 'OBLIVION':
+                            if NifOp.props.pad_bones:
+                                NifLog.warn("Using padbones on Oblivion export. Disable the pad bones option to get higher quality skin partitions.")
+                        if game in ('OBLIVION', 'FALLOUT_3'):
```

Comment:
Could simplify with if  `!=18` though this does give more precise messaging, will leave it up to you.

Reply:
Not sure what you mean?

Reply:
There are two code paths `if NifOp.props.max_bones_per_partition < 18:` and `if NifOp.props.max_bones_per_partition < 18`, which could be folded into one, i.e `if NifOp.props.max_bones_per_partition != 18:` however having both path allows for better user messaging. 

Its probably better the way it is.

Reply:
I've refactored this part to be more generic.